index.php

```diff
@@ -12,8 +12,10 @@
     for($q = 0; $q < count($food[$i]); $q++)
     {
         echo ' | ' . $food[$i][$q];
-        echo '---------------------------';
```

Comment:
1) add last line in row
2) format "array" declaration
3) add minuses on right way